### DISCUSSION TITLE
XYZ Chart Feedback

### DISCUSSION THREAD

**Discussion by mdvictor** [grafana/xyz-chart#44] [Announcements]
Hello everyone!

Our team at Grafana has released an alpha version of a 3D scatter plot panel. The XYZ Chart brings a new way of visualising your data, by plotting it in three dimensions and showing the relationship between variables. The chart is easy to set up and use: you can either let the panel automatically select the dimensions in `Auto` mode or decide which 3 variables should be applied in `Manual` mode. The panel is interactive in the sense that the plot can be rotated, panned and zoomed in and out while the datapoints can be hovered for more details.

![xyz-chart-demo](https://user-images.githubusercontent.com/36818606/221137395-0e8bb4d2-052a-4267-b429-461b6cbfe9f0.gif)

The XYZ Chart is, at the moment, in alpha and we are looking to extend it's functionality further. We have planned additional features for future releases (e.g: multiple series support, ability to change data point shape from circle to other primitives, multiple dimensions represented by data point size or colour hue, connected lines, 3D meshes, etc.) and with this discussion we aim to understand what other use cases we should consider as development continues and what other features you would like to see in the panel.

We are eager to hear your feedback and are looking forward to bringing more features into the XYZ Chart.

**Comment by black-ghost-off** [comment]
It is a very useful tool for research. Grafana is not only a system for monitoring resources, but also for visualizing various absolutely different data from different sources (such as information about the position of the drone in space, information from a large number of sensors in real time, and much more)
